### PR TITLE
Swift 3.0

### PR DESCRIPTION
**Pull request by codecaffeine**
Update the project with Swift 3.0 support

### CODE REVIEW

File: Pod/Classes/LayoutConvenience.swift [hunk 1]
```diff
@@ -20,23 +20,23 @@ public func activateVFL(format format: String, options: NSLayoutFormatOptions =
 }
 
 public extension UIView {
-    @available(*, deprecated, message="addSubviewsWithNoConstraints instead")
-    public func addSubviewWithNoConstraints(subview: UIView) {
+    @available(*, deprecated, message: "addSubviewsWithNoConstraints instead")
```
Comment (jgrandelli) [inline Pod/Classes/LayoutConvenience.swift]: This should be actually removed. No longer deprecated

File: Pod/Classes/TableViewHelpers.swift [hunk 2]
```diff
@@ -30,23 +30,23 @@ extension UITableView {
 }
 
 public enum ReusableCell<Cell> {
-    case Class(identifier: String)
-    case Nib(identifier: String, nibName: String, bundle: NSBundle?)
+    case `class`(identifier: String)
```
Comment (ryang1428) [inline Pod/Classes/TableViewHelpers.swift]: is this right? should this be `.class`?

Comment (jgrandelli) [inline Pod/Classes/TableViewHelpers.swift]: @codecaffeine thoughts on this? It shouldnt be `.class` but something is up. Is that apples new way of saying class

Comment (codecaffeine) [inline Pod/Classes/TableViewHelpers.swift]: >
“To use a reserved word as an identifier, put a backpack (\`) before and after it. For example, class is not a valid identifier, but \`class\` is valid. The backticks are not considered part of the identifier; \`x\` and x have the same meaning.”

Excerpt From: Apple Inc. “The Swift Programming Language (Swift 2.2).” iBooks. https://itun.es/us/jEUH0.l

**Comment by jgrandelli** [comment]
Tests passed locally on merged result

**Pull request merged**